src/components/augmentedReality/AugmentedReality.js

```diff
@@ -50,18 +50,24 @@ export const AugmentedReality = ({
   const [isModalVisible, setIsModalVisible] = useState(false);
   const selectedFilterId = filter.find((entry) => entry.selected)?.id;
 
-  useEffect(() => {
-    const { settings = {} } = globalSettings;
-
+  const supportAr = async () => {
```

Comment:
should be a `useCallback` i guess to avoid running multiple times various re-rendering?

Reply:
or couldn't this be placed directly in the `useEffect`?

Reply:
I prefer to write it as in `SettingsScreen.js` to have the same structure.